### ISSUE TITLE
Fetchmail does not fetch seen mails

### ISSUE THREAD

**Issue by NicolasGoeddel** [Mailu/Mailu#2995]
## Environment & Version

- Docker Compose version v2.20.0
- MailU version 2.0

## Description
When adding an external mail account to be fetched only seen mails get transferred. There is an option missing to also fetch all seen mails which I thought what would happen by default.

## Replication Steps
- Add a fetchmail account that consists of seen and unseen mails
- Wait for the cronjob to execute

## Observed behaviour
- Only unseen mails were fetched

## Expected behaviour
- All mails from all listed folders were fetched

## Logs
I don't think logs help here. They also contain of private information.

**Comment by ghostwheel42** [comment]
Are you sure. As far as I can see the option "[fetchall](https://www.fetchmail.info/fetchmail-man.html#disposal-options)" will enable receiving of seen emails:
https://github.com/Mailu/Mailu/blob/2.0/optional/fetchmail/fetchmail.py#L56

**Comment by NicolasGoeddel** [comment]
If you want to keep the mails on the server `fetchall` will not be set.
I want to keep the mails but also fetch them all.

However, I already overrode the`fetchmail.py` script so both `fetchall` and `keep` were in place. And now at every invokation all mails were put into my mailbox and now I have a lot of double entries. Also all mails are missing their original date and also are `unseen` again.
That was not was I was hoping for. I just wanted to transfer all mails from an external IMAP mailbox to my own, including their original date, their location in the folders and their `seen` status. It seems `fetchmail` is not able to do that.

**Comment by ghostwheel42** [comment]
Yes, this is not the idea behind the fetch feature.
I think "fetchmail" cannot keep track of what has been transferred already when you enable "keep" and" fetchall" at once.
It helps people wanting to keep a second email address with some other provide.
If you want to bulk-migrate your emails you can start `fetchmail` manually one or use `imapsync` or even thunderbird (drag and drop).

**Comment by NicolasGoeddel** [comment]
Yes, it looks like I misunderstood that completely. I will give `imapsync` a try.

Sorry for creating this unnecessary ticket.
